azure_ad_admin/icon_azure_ad_admin/util/get_user_info.py

```diff
@@ -3,13 +3,29 @@
 from komand.exceptions import PluginException
 
 
-def get_user_info(connection, user_id):
+def get_user_info(connection, user_id, logger):
     endpoint = f"https://graph.microsoft.com/v1.0/{connection.tenant}/users/{user_id}"
     headers = connection.get_headers(connection.get_auth_token())
 
-    result = requests.get(endpoint, headers=headers)
+    result = None
+    try:
+        result = requests.get(endpoint, headers=headers)
+    except Exception:
+        counter = 0; # We are going to try this 5 times and give up.
+        while counter < 5:
+            counter += 1
```

Comment:
```suggestion
        for counter in range(1, 6):
```
Takes care of the 5 count retry and incrementing for ya